app/js/models/todo.js

```diff
@@ -8,7 +8,7 @@ define([
     },
 
     changeCompleted: function () {
-      this.completed = !this.get('completed');
+      this.set({completed: !this.get('completed')});
```

Comment:
As a side note, the method name is not the best one one, `changeCompleted` to what? `toggleCompleted` is a way better name.

(this does not really matter as this method should not be here0